library/swift/EngineBuilder.swift

```diff
@@ -211,6 +225,17 @@ open class EngineBuilder: NSObject {
     return self
   }
 
+  /// Set the maximum number of connections to open to a single host.
```

Comment:
same as above - can we add a comment which would explain what the default maximum is (7)?

Reply:
There's not really a good reason - 7 is just in the range of normal defaults for this kind of thing. I did want to get in the habit of mentioning defaults in the docstrings though, which was why I mentioned it here.